turbinia/client.py

```diff
@@ -131,6 +131,29 @@ def get_turbinia_client(run_local=False):
     raise TurbiniaException(msg)
 
 
+def register_job_timeouts(dependencies):
+  """Registers a timeout for each job.
+
+  Args:
+    dependencies(dict): dependencies to grab timeout value from.
+  """
+  log.info('Registering job timeouts.')
+  timeout_default = 3600
+
+  job_names = list(job_manager.JobsManager.GetJobNames())
+  # Iterate through list of jobs
+  for job, values in dependencies.items():
+    if job not in job_names:
+      continue
+    timeout = values.get('timeout')
+    if type(timeout) != int:
```

Comment:
Infinitesimal nit -- I wouldn't even mention it since I missed it the first time, but since I had the other comment anyway: FYI the more idiomatic way to do this check is with `isinstance()`.

Reply:
Ah good to know/agree with doing it the idiomatic, updated to use isinstance instead.